sympy/matrices/common.py

```diff
@@ -2018,7 +2018,59 @@ def replace(self, F, G, map=False, simultaneous=True, exact=None):
         """
         return self.applyfunc(
             lambda x: x.replace(F, G, map=map, simultaneous=simultaneous, exact=exact))
+    
```

Comment:
```suggestion

```
I think that there is a trailing space error here for travis

Reply:
Done